tests/torch/ptq/test_wrap_model.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from torch import nn
+
+from nncf.torch.model_creation import wrap_model
+
+
+class TestArgumentModel(nn.Module):
+    def __init__(self, example_input) -> None:
+        super().__init__()
+        self._example_input = example_input
+
+    def forward(self, x):
+        assert x == self._example_input
+        return x
+
+
+class TestNonKeyWordArgumentsModel(nn.Module):
+    def __init__(self, example_input) -> None:
+        super().__init__()
+        self._example_input = example_input
+
+    def forward(self, x, y):
+        assert x == self._example_input[0]
+        assert y == self._example_input[1]
+        return x, y
+
+
+class TestKeyWordArgumentsModel(nn.Module):
+    def __init__(self, example_input) -> None:
+        super().__init__()
+        self._example_input = example_input
+
+    def forward(self, *, x, y):
+        assert x == self._example_input["x"]
+        assert y == self._example_input["y"]
```

Comment:
Execute this under `nncf.torch.dynamic_graph.context.no_nncf_trace` so that the `__eq__` nodes don't end up in the graph.

Reply:
done